development/docker-compose.yml

```diff
@@ -15,21 +15,21 @@ services:
     command: [ "/bin/sh", "-c", "while true; do sleep 30; done;" ]
 
   webdoc_cvp:
-    image: titom73/mkdocs:latest
+    image: squidfunk/mkdocs-material:9.0.12
     container_name: webdoc_cvp
     volumes:
       - ${PWD}/:/docs
     ports:
       - 8001:8000
     entrypoint: ""
-    command: ["sh", "-c", "cd ansible-cvp/ && pip install -r ansible_collections/arista/cvp/docs/requirements.txt && mkdocs serve --no-livereload --dev-addr=0.0.0.0:8000 -f mkdocs.yml"]
+    command: ["sh", "-c", "cd ansible-cvp/ && pip install -r ansible_collections/arista/cvp/docs/requirements.txt --upgrade && mkdocs serve --no-livereload --dev-addr=0.0.0.0:8000 -f mkdocs.yml"]
 
   webdoc_avd:
-    image: titom73/mkdocs:latest
+    image: squidfunk/mkdocs-material:9.0.12
     container_name: webdoc_avd
     volumes:
       - ${PWD}/:/docs
     ports:
       - 8000:8000
     entrypoint: ""
-    command: ["sh", "-c", "cd ansible-avd/ && pip install -r ansible_collections/arista/avd/docs/requirements.txt && mkdocs serve --no-livereload --dev-addr=0.0.0.0:8000 -f mkdocs.yml"]
+    command: ["sh", "-c", "cd ansible-avd/ && pip install -r ansible_collections/arista/avd/docs/requirements.txt --upgrade && mkdocs serve --no-livereload --dev-addr=0.0.0.0:8000 -f mkdocs.yml"]
```

Comment:
so maybe not for this PR ... I would challenge keeping this if we change the container. if you look at our CI pipeline we are actually doing some `sed` operation to remove the `cd ansible-avd`.

https://github.com/aristanetworks/ansible-avd/pull/2536/files#diff-6f0e9ef0c3a211590d88765ce7dee35c551f602cf76d5d67fe66828b6c34ae3cR102

A more general question is do we need to keep docker-compose at this stage - is anybody still using this docker-compose ?

Maybe we could start webdoc_avd as a service container for the job instead
https://docs.github.com/en/actions/using-containerized-services/about-service-containers